packages/site/src/pages/Article.tsx

```diff
@@ -17,7 +17,13 @@ import {
 } from '@myst-theme/jupyter';
 import { FrontmatterBlock } from '@myst-theme/frontmatter';
 
-export const ArticlePage = React.memo(function ({ article }: { article: PageLoader }) {
+export const ArticlePage = React.memo(function ({
+  article,
+  hide_all_footer_links,
+}: {
+  article: PageLoader;
+  hide_all_footer_links?: boolean;
```

Comment:
agonised over this a bit, setting on individual pages might be enough but the `frontmatter.design` code here seems to be prospective i.e. it's not possible to pass those options in on page frontmatter.